src/App.vue

```diff
@@ -35,6 +137,39 @@
   </div>
 </template>
 
+<script>
+export default {
+  data() {
+    return {
+      menu: false,
+    };
+  },
+
+  computed: {
+    version() {
+      return this.$store.getters.version;
+    },
```

Comment:
Is this even needed anymore?

Reply:
Ah yeah, that was to show the version in the navigation which has been removed. So I guess this can be removed, also.